config/routes.js

```diff
@@ -1,9 +1,12 @@
 const ensureAuthenticated = require('../middleware/ensure-auth');
 
+const env = process.env.NODE_ENV || 'development';
+const config = require('./config')[env];
```

Comment:
i think i originally passed config into the call to routes, but i don't think it matters

Reply:
Would it be better to add the `config` object as a function param in the routes? would avoid this extra require statement and NODE_ENV check

Reply:
yes it would.

Reply:
small change so we might as well do it.